.ci/run_nccl_tests.sh

```diff
@@ -1,45 +1,44 @@
 #!/bin/bash -leE
-
-SCRIPT_DIR="$( cd "$(dirname "$0")" ; pwd -P )"
+GH=${GH:-0}
+if [ "${GH}" -eq 1 ]; then
+    GH_FOLD="::group::"
```

Comment:
Please add a comment that this's GH Actions specific formatting macros

Reply:
Added